src/logic/safe/store/actions/createTransaction.ts

```diff
@@ -76,8 +77,13 @@ export const createTransaction = (
     dispatch(push(`${SAFELIST_ADDRESS}/${safeAddress}/transactions`))
   }
 
-  const ready = await onboardUser()
-  if (!ready) return
+  try {
+    const ready = await onboardUser()
```

Comment:
This is where it was throwing the "signatures too short" exception.

Reply:
Are you sure? I've seen this error before but it was from the backend when the signature was bad.

`onboardUser` doesn't check for signatures at all, if has checks for the correct network/balance/etc, read more about it here: https://www.blocknative.com/onboard

Reply:
I'm not sure it even throws

Reply:
That's what the stack trace was showing 🤷
After catching it, it logs 803 as on the screenshot above.

Reply:
But it's not the only place that logs 803. Also, it says `request failed with code 422` on the screenshot, there's a problem somewhere with generating the signature

Reply:
True, let me double-check.

Reply:
You're right, it was actually failing on `getErrorMessage`, ironically.

Reply:
I'm now logging both 422 and the signatures error:

<img width="1116" alt="Screenshot 2021-06-15 at 14 45 01" src="https://user-images.githubusercontent.com/381895/122058039-85290a00-cdeb-11eb-9bfa-6738f77e01dd.png">

Reply:
How do you reproduce it?

Reply:
By setting the nonce to lower than the previous.